### ISSUE TITLE
double_quoted_strings does not permit space escape sequence

### ISSUE THREAD

**Issue by optiz0r** [puppetlabs/puppet-lint#16]
## Describe the Bug

The `double_quoted_strings` raises a false positive for a double quoted string which contains the `\s` escape sequence. This sequence is only valid within double-quoted strings, not in single quoted strings, so the type of quotes cannot be changed, and a string using this sequence may legitimately not contain any variables.

## Expected Behavior

No lint warning to be raised

## Steps to Reproduce

Include the following code in a manifest: `"\s"`

Example:

```
split(strip($foo),"\s")
```
## Environment
 - Version 2.4.2
 - Platform CentOS, puppet 6

## Additional Context

https://github.com/rodjek/puppet-lint/blob/master/lib/puppet-lint/plugins/check_strings/double_quoted_strings.rb#L7 should include `\\s` as an additional case for a valid string to not fail the check.
